.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Why is this needed?

Reply:
Not sure why it is needed (or how it got there..) but removing this causes the following error when running yarn install
`
# This file contains the result of Yarn building a package (jbr-monorepo@workspace:.)
# Script name: postinstall


> build:ts
> tsc

.yarn/cache/@comunica-actor-abstract-parse-npm-2.8.2-7c3e8e4934-ef366d5f4c.zip/node_modules/@comunica/actor-abstract-parse/lib/ActorAbstractParse.d.ts(3,31): error TS7016: Could not find a declaration file for module 'readable-stream'. '/home/reschauz/projects/jbr-yarn-lock-reset/.yarn/cache/readable-stream-npm-4.4.2-ac0d67d9ca-6f4063763d.zip/node_modules/readable-stream/lib/ours/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/readable-stream` if it exists or add a new declaration (.d.ts) file containing `declare module 'readable-stream';`
.yarn/cache/rdf-parse-npm-2.3.2-1433dc1169-61f32aed3d.zip/node_modules/rdf-parse/lib/RdfParser.d.ts(4,26): error TS7016: Could not find a declaration file for module 'readable-stream'. '/home/reschauz/projects/jbr-yarn-lock-reset/.yarn/cache/readable-stream-npm-4.4.2-ac0d67d9ca-6f4063763d.zip/node_modules/readable-stream/lib/ours/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/readable-stream` if it exists or add a new declaration (.d.ts) file containing `declare module 'readable-stream';`
.yarn/cache/rdf-parse-npm-2.3.2-1433dc1169-61f32aed3d.zip/node_modules/rdf-parse/lib/index.d.ts(3,42): error TS2307: Cannot find module 'rdf-js' or its corresponding type declarations.
`

Reply:
Node 18 doesn't ship with yarn 3.

But I guess it doesn't hurt to have for people who use yarn 3.